provider/resources.go

```diff
@@ -421,6 +429,25 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
+	prov.MustComputeTokens(tfbridgetokens.KnownModules("openstack_", "Index", []string{
```

Comment:
Applied auto-token mapping here, would like confirmation that this looks okay. Pretty sure the modules are all supposed to be one word based on how they appear in the rest of the file, and "Index" suffices for the main mod (there wasn't one defined)

Reply:
```suggestion
	prov.MustComputeTokens(tfbridgetokens.KnownModules("openstack_", "index", []string{
```

Our normal "main" moduled is `"index"` (lowercase).

I would delete some resources to make sure it's working as expected, but otherwise it looks good.